src/Sulu/Bundle/DocumentManagerBundle/Routing/Loader/VersionRouteLoader.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\DocumentManagerBundle\Routing\Loader;
+
+use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * This route loader should only load the containing routes if the versioning feature in the document manager is enabled.
+ */
+class VersionRouteLoader extends Loader
+{
+    /**
+     * @var bool
+     */
+    private $enabled;
+
+    public function __construct($enabled)
+    {
+        $this->enabled = $enabled;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load($resource, $type = null)
+    {
+        if (!$this->enabled) {
+            return new RouteCollection();
+        }
+
+        return $this->import($resource, 'rest');
```

Comment:
Has to be rest, because some attributes are not available otherwise. Don't know a better solution than making a `version_rest` route loader...